tools/validators/instance_validator/validate/telemetry.py

```diff
@@ -77,8 +77,12 @@ def _parse_data(self, data):
     version = json_object[VERSION]
     timestamp = json_object[TIMESTAMP]
     points = {}
+    if POINTS not in json_object.keys():
```

Comment:
As mentioned above, I like the idea of making this as generic as possible by not assuming that 'points' is the top-level key for the JSON telemetry values.

Instead we can validate this kind of structural convention at the entity_instance level and then rely on the YAML translation to tell us where to find the points in the JSON payload.

Reply:
created a todo for the rest,
this check is diferent, often we are receiving from live data error messages with strings (logs) instead of points. 
this check is necessary here